src/setfit/modeling.py

```diff
@@ -95,8 +99,21 @@ def _from_pretrained(
         if model_head_file is not None:
             model_head = joblib.load(model_head_file)
         else:
-            model_head = LogisticRegression()
-        return SetFitModel(model_body=model_body, model_head=model_head)
+            if multi_target_strategy:
```

Comment:
nit: we generally prefer to be explicit :)
```suggestion
            if multi_target_strategy is not None:
```